src/components/AddItemForm/AddItemForm.jsx

```diff
@@ -0,0 +1,267 @@
+                    <ComboBox
+                      options={ITEM_CATEGORY}
+                      value={item.category}
+                      onChange={(e) => setItem({ ...item, category: e })}
+                    />
+                  </div>
+                </div>
+                <div>
+                  <TextArea
+                    name="description"
+                    errors={errors?.description}
+                    {...register('description')}
+                  >
+                    Description
+                  </TextArea>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="hidden sm:block py-2" aria-hidden="true">
+          <div className="">
+            <div className="border-t border-secondary" />
+          </div>
+        </div>
+
+        <div className="md:grid md:grid-cols-4 md:gap-6">
+          <div className="md:col-span-1">
+            <div className="mx-4 py-3">
+              <h3 className="text-xl font-bold text-primary">
+                Contact Information
+              </h3>
+              <p className="mt-1 text-sm text-secondary font-semibold">
+                Use permanent contact information to receive emails from potential buyers.
+              </p>
+            </div>
+          </div>
+          <div className="mt-5 md:col-span-3 md:mt-0 pr-2 pb-4 md:pl-0 pl-2">
+            <div className="overflow-hidden shadow sm:rounded-md">
+              <div className="bg-primary px-4 py-5 sm:p-6">
+                <div className="grid grid-cols-6 gap-6">
+                  <div className="col-span-6 sm:col-span-4">
+                    <Input
+                      name="email"
+                      type="text"
+                      errors={errors?.email}
+                      {...register('email')}
+                    >
+                      Email address
+                    </Input>
+                  </div>
+
+                  <div className="col-span-6 sm:col-span-4">
+                    <Input
+                      name="country"
+                      type="text"
+                      errors={errors?.country}
+                      {...register('country')}
+                    >
+                      County
+                    </Input>
+                  </div>
+
+                  <div className="col-span-6 sm:col-span-4">
+                    <Input
+                      name="city"
+                      type="text"
+                      errors={errors?.city}
+                      {...register('city')}
+                    >
+                      City
+                    </Input>
+                  </div>
+                </div>
+              </div>
+              <div className="bg-primary bg-opacity-25 px-4 py-3 text-right sm:px-6">
+                <SubmitButton buttonText="Add New Item" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </form>
+    </div>
+  );
+}
```

Comment:
We will have users email since they will be authenticated, so no need to ask for it again. 

I am contemplating on country and the city. Should the user have to give those every time they put up an item? That sounds tedious. Maybe we should ask for users location data in sign up, or make the user add it in their profile when after they sign up? What do you think ?

Reply:
I suggest adding them with the sign up form

Reply:
agree on this point

Reply:
Create an issue for that please.